spec/app_spec.rb

```diff
@@ -154,28 +157,20 @@ def app
     end
 
     it "should validate the position, cannot play on already played position" do
-      body1 = {
+      body = {
         "game_id":@game_id,
         "player":1,
         "position":3
       }.to_json
 
-      body2 = {
-        "game_id":@game_id,
-        "player":2,
-        "position":3
-      }.to_json
       expected_response = {"error": {"position": "already occupied"}}.to_json
-      @datastore.store(@game_id, {})
-
-      post '/play', body1, @request_headers
-      expect(last_response).to be_ok
-
-      puts last_response.errors
+      @web_game.store({@game_id=> {'state': ['','','X','','','','','','']}})
 
-      post '/play', body2, @request_headers
+      post '/play', body, @request_headers
+      response = JSON.parse(last_response.body)
 
       expect(last_response).to be_ok
+      expect(response).to include('error')
```

Comment:
This may be redundant considering you are asserting the entire response body to be the expected response on the next line.